tests/test_cli.py

```diff
@@ -204,21 +204,20 @@ def test_command_add_objective(workdir, mocker):
     (['--trunk-model-key', 'e'], r'The --head-model-key option is required', 2)
 ])
 def test_command_add_composite_traintuple(mocker, workdir, params, message, exit_code):
-    with mock_client_call(mocker, 'add_composite_traintuple', response={}) as m:
+    with mock_client_call(mocker, 'add_composite_traintuple', response={}):
```

Comment:
Looks like nothing is returned from `m`, that's why I removed it

Reply:
Actually there is something in m, however the issue is with the use of `with`. `mock_client_call` cannot be used as a context manager (see https://github.com/SubstraFoundation/substra/pull/41). You should use `m = mock_client_call(...)` and keep the assertion at the end.

Be careful through, half of the parametrization cases will trigger an error and therefore the mocker won't be called. You may need to either add an if before the assertion or split the test in 2, one for calls that success and the other for calls that fail.